docs/Examples.md

```diff
@@ -83,11 +83,15 @@ peer.
 
 ```
 $ kubectl get pods
-NAME                               READY   STATUS    RESTARTS   AGE
-splunk-cm-cluster-master-0         1/1     Running   0          29s
-splunk-example-indexer-0           1/1     Running   0          29s
-splunk-operator-7c5599546c-wt4xl   1/1     Running   0          14h
+NAME                                       READY   STATUS    RESTARTS    AGE
+splunk-cm-cluster-master-0                  1/1     Running   0          29s
+splunk-default-monitoring-console-0         1/1     Running   0          15s
```

Comment:
Nit: Do we need to mention splunk-<namespace>-monitoring-console-0 vs splunk-default-monitoring-console-0?

Reply:
Thought about this, but don't want to create a confusion with `splunk--monitoring-console-0`, so by default assuming we are talking about default namespace in the example.md